locust_plugins/csvreader.py

```diff
@@ -4,13 +4,13 @@
 class CSVReader:
     "Read test data from csv file using an iterator"
 
-    def __init__(self, file):
+    def __init__(self, file, dialect="excel", *args, **kwargs):
```

Comment:
I dont think changing the default make sense (people might be confused/surprised by this)